core/src/main/java/org/mule/runtime/core/DefaultMuleContext.java

```diff
@@ -266,9 +266,9 @@
       }
     });
 
-    // Log dropped events/errors rather than blow up which causes cryptic timeouts and stack traces.
-    Hooks.onErrorDropped(error -> logger.error("ERROR DROPPED UNEXPECTEDLY " + error));
-    Hooks.onNextDropped(event -> logger.error("EVENT DROPPED UNEXPECTEDLY " + event));
+    // Log dropped events/errors
+    Hooks.onErrorDropped(error -> logger.debug("ERROR DROPPED " + error));
```

Comment:
Yes, there is logging en EventContext anyway, for the majority of use cases.  We cannot leave these `error` because when using `timeout` dropping the event is valid, and it's then replaced with another one.